src/Json.Schema.Validation.UnitTests/TestData/ComprehensiveInstanceFile-validation-expected.sarif

```diff
@@ -1074,20 +1073,17 @@
                 "level": "error"
               }
             }
-          ],
-          "properties": {
-            "Comments": "Command line tool to validate a JSON instance against a JSON Schema"
-          }
+          ]
         }
       },
       "invocations": [
         {
-          "startTimeUtc": "2020-07-28T15:22:49.841Z",
-          "endTimeUtc": "2020-07-28T15:22:50.230Z",
+          "startTimeUtc": "2022-04-22T20:26:56.699Z",
+          "endTimeUtc": "2022-04-22T20:26:56.802Z",
```

Comment:
it is not necessary to checkin this but I think we can keep it, it contains info when it was last updated.

Reply:
pass in the test, the option to remove nondeterministic properties.

Reply:
I checked that is only in SDK, in this Json.Schema.Validation.Cli which we are testing don't have that option.
`
int exitCode = Program.Main(new string[]
            {
                "--schema-file-path", SchemaFilePath,
                "--log-file-path", ActualOutputFilePath,
                "--instance-file-path", InstanceFilePath,
                //"--remove", DataToRemove,
            });
`